webapp/src/components/Navbar/Navbar.css

```diff
@@ -112,6 +112,47 @@
   display: none;
 }
 
+.Navbar .hamburger {
+  color: var(--primary);
+  margin-right: 0.5em;
+}
+
+.Navbar .hamburger-wrapper {
+  font-family: var(--font-family);
+  text-transform: uppercase;
+  font-weight: bold;
+  color: white;
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  cursor: pointer;
+  padding-bottom: 5px;
+}
+
+.Navbar .sidebar {
+  background: linear-gradient(#1f2333, #0d0f17) !important;
```

Comment:
I think both of these colors are repeated quite a bit throughout the app, we may want to add them to `variables.css`

_Edit_: `#1f2333` is already there as `--background` :D

Reply:
✅  done (724b40e)